src/commands/echo.ts

```diff
@@ -11,6 +11,9 @@ import {
   strikethrough,
   underscore,
 } from "@discordjs/builders";
+import { fromEnv } from "../config";
+
+const config = fromEnv();
```

Comment:
Also, commands shouldn't be concerned with `config`. If the required role id cannot be found, the command shouldn't be registered. And that should be done in `updateCommands`.

Reply:
Noted with thanks.